wolfhsm/wh_client_crypto.h

```diff
@@ -372,7 +370,7 @@ int wh_Client_AesCmacVerify(Cmac* cmac, const byte* check, word32 checkSz,
  * @return int Returns 0 on success, or a negative error code on failure.
  */
 int wh_Client_CmacCancelableResponse(whClientContext* c, Cmac* cmac,
-    uint8_t* out, uint32_t* outSz);
+    uint8_t* sig, uint16_t* outSz);
```

Comment:
never heard cmac parameters referred to as a signature/hash... was this deliberate? `in` and `out` seemed entirely appropriate here, and align with the wolfCrypt arg names too....maybe `inData`/`macToVerify` and `outMac` if you want more specificity?

Not critical, but if desired, can change in subsequent PR